package.json

```diff
@@ -9,7 +9,7 @@
   "type": "module",
   "scripts": {
     "dev": "next dev --port=6060",
-    "build": "next build",
+    "build": "npx ts-node-esm src/scripts/build/createLocationAutocompleteData/index.ts && next build",
```

Comment:
I don't think we should use `npx` here, we should just install `ts-node-esm` as a dev dependency. It's better to pin and let dependabot et al update vs. not knowing what version was used etc.

Reply:
`ts-node-esm` is packaged with `ts-node` dependency.

```sh
cat package.json | jq '.devDependencies'

{ "ts-node": "^10.9.1" }
```

```sh
ls -l ./node_modules/.bin

acorn -> ../acorn/bin/acorn
ts-node -> ../ts-node/dist/bin.js
ts-node-cwd -> ../ts-node/dist/bin-cwd.js
ts-node-esm -> ../ts-node/dist/bin-esm.js
ts-node-script -> ../ts-node/dist/bin-script.js
ts-node-transpile-only -> ../ts-node/dist/bin-transpile.js
ts-script -> ../ts-node/dist/bin-script-deprecated.js
tsc -> ../typescript/bin/tsc
tsserver -> ../typescript/bin/tsserver
```

Reply:
That’s right! Calling `ts-node-esm` directly now instead of through `npx`: [2c46792](https://github.com/mozilla/blurts-server/pull/3186/commits/2c467928164054628ac090b3bde1e130ffad3e79).